tests/site/click.html

```diff
@@ -48,7 +48,7 @@
                 window.ondblclick = function(event) {
                     results.testdoubleclick = [event.clientX, event.clientY];
                 };
-                window.oncontextmenu = function() {
+                window.oncontextmenu = function(event) {
```

Comment:
It actually worked in PhantomJS due to an implicit `event` variable in Webkit (a125fd1).

The issue was in SlimerJS where Gecko uses the function definition to determine the variable name.